src/common/Logging/Log.cpp

```diff
@@ -106,18 +106,18 @@ void Log::Initialize()
 
     /// Open specific log files
     logfile = openLogFile("LogFile", "LogTimestamp", "w");
-    InitColors(sConfigMgr->GetStringDefault("LogColors", ""));
+    InitColors(sConfigMgr->GetStringDefault("LogColors", "", false));
```

Comment:
`LogColors` is have in `worldserver.conf` and `authserver.conf`
[../worldserver.conf.dist](https://github.com/azerothcore/azerothcore-wotlk/blob/334a0f3e2c5c43be448be8dbcfc9d6dffeb64266/src/server/worldserver/worldserver.conf.dist#L636)
[../authserver.conf.dist](https://github.com/azerothcore/azerothcore-wotlk/blob/334a0f3e2c5c43be448be8dbcfc9d6dffeb64266/src/server/authserver/authserver.conf.dist#L161)